access.go

```diff
@@ -478,12 +478,29 @@ func (s *Server) handleDeviceRequest(w *Response, r *http.Request) *AccessReques
 }
 
 func (s *Server) handlePlatformRequest(w *Response, r *http.Request) *AccessRequest {
-	// get client authentication
-	auth := GetClientAuth(w, r, s.Config.AllowClientSecretInParams)
-	if auth == nil {
+	auth, err := CheckBasicAuth(r)
+	if err != nil {
+		w.InternalError = err
```

Comment:
I think you need to add `w.SetError(E_INVALID_REQUEST, "")` like on the `GetClientAuth` function

Reply:
okay